dvc/command/remote.py

```diff
@@ -108,6 +108,34 @@ def run(self):
         return 0
 
 
+class CmdRemoteRename(CmdRemote):
+    def run(self):
+        conf = self.config.load_one(self.args.level)
+        self._check_exists(conf)
+
+        all_config = self.config.load_config_to_level("all")
+        if self.args.new in all_config.get("remote", {}):
+            raise ConfigError(
+                "Rename failed.Remote name {} already exists.".format(
```

Comment:
```suggestion
                "Rename failed. Remote name '{}' already exists.".format(
```

This could also be an f-string now instead of using `format()`